src/options/mkoptions.py

```diff
@@ -346,8 +346,19 @@ def die(msg):
     sys.exit('[error] {}'.format(msg))
 
 
-def perr(filename, lineno, msg):
-    die('parse error in {}:{}: {}'.format(filename, lineno + 1, msg))
+def perr(filename, msg, option_or_alias = None):
+    msg_prefix  = ''
+    if option_or_alias:
+        if isinstance(option_or_alias, Option):
+            msg_prefix = 'option '
+            if option_or_alias.name:
+                msg_prefix = "{} '{}' ".format(msg_prefix, option_or_alias.name)
+            else:
+                msg_prefix = "{} '{}' ".format(msg_prefix, option_or_alias.long)
+        else:
+            assert isinstance(option_or_alias, Alias)
+            msg_prefix = "alias '{}' ".format(option_or_alias.long)
+    die('parse error in {}: {}{}'.format(filename, msg, msg_prefix))
```

Comment:
The `msg_prefix` looks more like a `msg_suffix` to me :)

Reply:
Haha true, it was a prefix when I started but then made a suffix out of it :-)